config_management/domains/views.py

```diff
@@ -0,0 +1,70 @@
+from django.shortcuts import render, redirect, HttpResponse
+from django.views.generic import View, TemplateView
+from .models import DomainName
+# from .forms import AddDomainForm
+from django.http.response import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+import json
+
+# Create your views here.
+
+def list_all_domains(request):
+    if request.method == 'GET':
+
+        temp = DomainName.objects.all()
+        domain_list = []
+        for i in temp:
+            domain_list.append((i.name, i.id))
+
+        args = {'listing all domains':'ok', 'status_code':200, 'list of all domains':domain_list}
+        return JsonResponse(args)
+        # return HttpResponse('OK 200')
+    else:
+        return JsonResponse({'error':'method not supported'})
+
+def list_one_domain(request, search_d_id):
+    if request.method == 'GET':
+
+        try:
+            temp = DomainName.objects.get(id=search_d_id)
+        except:
+            return JsonResponse({'domain does not exist':'sorry boi, the requested domain does not exist'})
```

Comment:
If it is an error, shouldn't error response structure be same at all places?
[Comment for all other places as well]